pep-0480.txt

```diff
@@ -373,46 +388,78 @@ across multiple machines of each developer.
 
 __ https://en.wikipedia.org/wiki/Transparency_%28human%E2%80%93computer_interaction%29
 
+Therefore, an easy-to-use signing solution is RECOMMENDED for developers. One
+solution is to use ephemeral keys, where the private key is only kept in memory
+(and not stored on disk) for as long as needed to produce the signature and is
+then safely discarded. This removes any long-term challenge of protecting the
+key by doing away with the key in the first place.
+
+`Sigstore`__ provides a way to create and use such ephemeral keys and maintain
+a chain of trust between the signature made on a distribution and the publicly
+available information stored on its transparency log. It uses certificates
+issued by ts certificate authority (Fulcio) to prove a signing key was valid
+when a signature was made. In-depth discussion of sigstore's architecture is
+beyond the scope of this document and for more information one may refer to
+https://sigstore.dev
```

Comment:
This is the Appendix, and I believe we should go into enough details about how the proposed solution would work here, the same way we described how TUF works in PEP 458.